CCFatigue_modules/5_DammageSummation/Harris/miner_harris.py

```diff
@@ -0,0 +1,210 @@
+""" CCFatigue - Module 5 - miner_harris.py
+This code takes in SNC + CYC data and output DAS data
+
+Harris diagram is described in Tassos red book p. 108 - the original papers are:
+[1] https://www.sciencedirect.com/science/article/pii/0142112394904782?via%3Dihub
+    (https://doi.org/10.1016/0142-1123(94)90478-2)
+[2] https://www.sciencedirect.com/science/article/pii/S0266353897001218?via%3Dihub
+    (https://doi.org/10.1016/S0266-3538(97)00121-8)
+"""
+
+import os
+import sys
+import math
+import numpy as np
+import pandas as pd
+from scipy import stats
+
+SRC_DIR = os.path.dirname(os.path.realpath(__file__))
+DATA_DIR = os.path.join(SRC_DIR, "..", "..", "..", "Data")
+HARRIS_MOD3_DIR = os.path.join(SRC_DIR, "..", "..", "3_CLD", "Harris")
+
+INPUT_SNC_FILENAME = "SNC_date_harris_input.csv"
+INPUT_SNC_FILE = os.path.join(DATA_DIR, INPUT_SNC_FILENAME)
+INPUT_CYC_FILENAME = "CYC_date_harris_ccinput.csv"
+INPUT_CYC_FILE = os.path.join(DATA_DIR, INPUT_CYC_FILENAME)
+
+OUTPUT_CSV_FILENAME = "DAS_harris.csv"
+OUTPUT_CSV_FILE = os.path.join(DATA_DIR, OUTPUT_CSV_FILENAME)
+
+# staticvalue.txt => constants
+UCS = 27.1
+UTS = 27.7
+
+# Factor loop
+FACTOR_START = 1.0
+FACTOR_STOP = 0.9
+FACTOR_STEP = -1.0
+
+
+def calculate_normalized_alternating_stress(
+    f: float, m: float, c: float, u: float, v: float
+):
+    """
+    Calculate the normalized alternating stress component
+    Equation 1 p9 Ref [2]
+    Inputs:
+    - f: function of the laminate tensile strength
+    - m: normalized mean stress
+    - c: normalized compression stress
+    - u: shape of the right (predominatly tensile) wings of bell-shaped curve
+    - v: shape of the left (predominatly compressive) wings of bell-shaped curve

```

Comment:
Excellent, you've got definitions ! Make sure f u v are defined everywhere in these scripts + in the variables_dictionary